src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -60,111 +63,124 @@
 @click.command("save")
 @click.option("--phylo-run-id", type=int, required=True)
 @click.option("sequences_fh", "--sequences", type=click.File("w"), required=False)
-@click.option("selected_fh", "--selected", type=click.File("w"), required=False)
-@click.option("metadata_fh", "--metadata", type=click.File("w"), required=False)
-@click.option("builds_file_fh", "--builds-file", type=click.File("w"), required=True)
-@click.option(
-    "county_sequences_fh", "--county-sequences", type=click.File("w"), required=False
-)
 @click.option(
-    "county_metadata_fh", "--county-metadata", type=click.File("w"), required=False
+    "selected_fh", "--selected", type=click.File("w", lazy=False), required=True
```

Comment:
lazy=False means we'll always write the include.txt file, whether it's empty or not.